packages/components/src/components/ScalarListbox/ScalarListbox.vue

```diff
@@ -16,13 +16,13 @@ import ScalarListboxOption from './ScalarListboxItem.vue'
 import type { Option } from './types'
 
 type SingleSelectListboxProps = {
```

Comment:
I should maybe make this two components like I did for combobox, it would prevent this props issue

Reply:
We can fix the typescript with some overloading I think